src/dashboard/apps/auth/factories.py

```diff
@@ -0,0 +1,27 @@
+"""Dashboard auth factories."""
+
+import factory
+from django.contrib.auth import get_user_model
+
+
+class UserFactory(factory.django.DjangoModelFactory):
+    """Factory class for creating a “standard” user."""
+
+    username = factory.Faker("user_name")
+    password = factory.django.Password("foo")
+
+    class Meta:  # noqa: D106
+        model = get_user_model()
+
+
+class AdminUserFactory(UserFactory):
+    """Factory class for creating a superuser."""
+
+    username = factory.Faker("superadmin")
```

Comment:
I cannot find this Faker provider. Is it required? Or can we use the same one as the `UserFactory`?